src/lib/runtime_config/runtime_config.ml

```diff
@@ -8,418 +8,282 @@ let yojson_strip_fields ~keep_fields = function
   | json ->
       json
 
-let yojson_rename_fields ~alternates = function
```

Comment:
I think you reverted the runtime config JSON PR by accident. Can you  revert e2cd4f8f2ed76489f38e334fa703ceb8effe83e7 please.